src/containers/lists/BidList/BidList.tsx

```diff
@@ -43,17 +47,22 @@ const BidList: React.FC<BidListProps> = ({
 	isAccepting,
 	isLoading,
 	highestBid,
+	isAcceptBidEnabled = false,
 }) => {
 	//////////////////
 	// Data & State //
 	//////////////////
 	const [blockNumber, setBlockNumber] = useState<number>();
 	const [isAcceptBidModal, setIsAcceptBidModal] = useState(false);
 	const [acceptingBid, setAcceptingBid] = useState<Bid | undefined>(undefined);
-	const { library } = useWeb3React<Web3Provider>();
+	const { library, account } = useWeb3React<Web3Provider>();
 
 	// Sort bids by date
 	const sortedBids = sortBidsByTime(bids);
+	const filteredBids = sortedBids.filter(
+		(bid) => bid.bidder.toLowerCase() !== account?.toLowerCase(),
+	);
```

Comment:
tiny nit: would be more efficient to only do the filtering if `isAcceptBid` is enabled - not a big deal as I don't imagine the lists will get that long, but something to consider!